frontend/server/src/main/java/com/amazonaws/ml/mms/util/NettyUtils.java

```diff
@@ -236,6 +239,42 @@ public static int getIntParameter(QueryStringDecoder decoder, String key, int de
         }
     }
 
+    public static Map<String, String> getAcceptHeader(HttpMessage message) {
```

Comment:
1. I think SM only care about "Accept" header, other headers are not relevant.
2. since inputData already takes a map, why not put all headers in it?

Reply:
I just wanted to handle all the accept headers. SM can use only Accept header for now and expand it to others as and when they need to.